main/tests.py

```diff
@@ -76,4 +78,59 @@ def test_dog_owner_homepage_is_visible_for_dog_owner(self, client, create_dog_ow
         client.force_login(user=create_dog_owner_user.user)
         response = client.get("/homepage/")
         assert response.status_code == 200
-        assert list(response.context['daycares']) == list(DayCare.objects.all())
+        assert set(response.context['day_care_queryset']) == set(DayCare.objects.all())
+
+
+@pytest.mark.django_db
+class TestDogOwnerHomePageView:
+    def test_search_daycare_present_only_available_daycares_on_specific_dates(self, client, create_dog_owner_user):
+        test_day_care = DayCare.create(username='testuser', email='test@email.com', password='test_password',
```

Comment:
didn't we add a bunch of fixtures to conftest so we won't need this anymore?

Reply:
The reason I didnt use the fixture from the confest is because this is a speacil case of day-care with capcity=1.

Reply:
What is the difference? how much more work do you need to do if you use the fixture?

Reply:
Just one for loop for decreasing the capcity from 50 to 0 (that means to create 50 orders to fulfill the capacity ) 
Another option is to change the capacity in the fixture . @Omeramsc

Reply:
Fixed.